BraveWallet/Crypto/CryptoView.swift

```diff
@@ -11,7 +11,7 @@ import Introspect
 import BraveUI
 
 public struct CryptoView: View {
-  var walletStore: WalletStore
+  @ObservedObject var walletStore: WalletStore
   @ObservedObject var keyringStore: KeyringStore
   
```

Comment:
if walletStore is also @ObservedObject will a change in `keyringStore` also a change in `walletStore`?

Reply:
Nope, `ObservableObject` does not hook up any child `ObservableObject ` by default, you have to do so yourself if you want that behaviour